deploy_wms-assistent.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#Please add variables as environmental variables in Travis settings for the project when using this script.
+#These lines should also be added in the "script block" of travis.yml.
+#- export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+#- echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$PR, BRANCH=$BRANCH"
+set -e
+BRANCH=$1
+FILENAME=forvaltningsportal_wms-assistent_$BRANCH.tar.gz
+echo "Making archive..."
+tar --directory=wms-assistent -zcf $FILENAME .
+echo "Deploying..."
+if [ "${BRANCH}" == "master" ]
+ then
+  sshpass -p $scp_pass scp -o StrictHostKeyChecking=no $FILENAME $scp_user@$scp_dest2
+  sshpass -p $scp_pass scp -o StrictHostKeyChecking=no $FILENAME $scp_user@$prod_dest2
```

Comment:
Da denne ser ut til å pakkes til egen zip-fil, så foreslår jeg å endre til "scp_dest3". Når jeg får litt bedre tid, så skal jeg prøve om jeg ikke får laget et litt mer intelligent deployment-script for portalen, slik at vi kan sende alt til samme sted.

Reply:
Done.